source/_app/pages/scheduled.js

```diff
@@ -122,7 +122,7 @@ app.scheduled = {
       },
       eventRender: function(event, element) {
         //Show tooltip when hovering over an event title
-        var toolTipContent = '<strong>' + event.title + '</strong><br/>' + moment(event.start).format('MMMM d, YYYY') + ' ' + moment(event.start).format('h:mm a');
+        var toolTipContent = '<strong>' + event.title + '</strong><br/>' + moment(event.start).format('MMMM D, YYYY') + ' ' + moment(event.start).format('h:mma');
```

Comment:
h:mma formats as 1.30am 
I've used 12 hour because we're entering the dates in 12 hour time and changing it once its added would be confusing for the user. Adding something at 1.30pm and then seeing 13:30 when the page reloads (see https://github.com/digirati-co-uk/elife-monitoring-dashboard-frontend/pull/18)